lucet-runtime/lucet-runtime-internals/src/instance/signals.rs

```diff
@@ -244,14 +239,7 @@ extern "C" fn handle_signal(signum: c_int, siginfo_ptr: *mut siginfo_t, ucontext
                 thunk();
                 true
             }
-        };
-
-        if switch_to_host {
-            // we must disable termination so no KillSwitch may fire in host code.
-            inst.kill_state.disable_termination();
```

Comment:
Talked about this here: https://github.com/bytecodealliance/lucet/pull/429#discussion_r382876336

Noting for posterity, since that discussion got orphaned via new changes.